near-contract-standards/src/fungible_token/core_impl.rs

```diff
@@ -209,7 +213,7 @@ impl FungibleToken {
 
         // Get the unused amount from the `ft_on_transfer` call result.
         let unused_amount = match env::promise_result(0) {
-            PromiseResult::NotReady => unreachable!(),
+            PromiseResult::NotReady => env::abort(),
```

Comment:
@evgenykuzyakov I remember you defended how this is supposed to be `unreachable` before. What do you think?

Reply:
disregard, i didn't see this farther down: https://github.com/near/near-sdk-rs/pull/527/files#diff-28f92d462ad3e2a8554f6a34935fbf5ba6f7dcfbc615546eff3741d4356adb1bR492